packages/protocol/contracts/stability/Exchange.sol

```diff
@@ -107,17 +107,95 @@ contract Exchange is
    * @return The amount of buyToken that was transfered
    * @dev This function can be frozen using the Freezable interface.
    */
+  function sell(uint256 sellAmount, uint256 minBuyAmount, bool sellGold)
+    external
+    onlyWhenNotFrozen
+    updateBucketsIfNecessary
+    nonReentrant
+    returns (uint256)
+  {
+    return _sell(sellAmount, minBuyAmount, sellGold);
+  }
+
+  /**
+   * @dev Deprecated. Exchanges sellAmount of sellToken in exchange for at
+   * least minBuyAmount of buyToken.
+   * Requires the sellAmount to have been approved to the exchange
+   * @param sellAmount The amount of sellToken the user is selling to the exchange
+   * @param minBuyAmount The minimum amount of buyToken the user has to receive for this
+   * transaction to succeed
+   * @param sellGold `true` if gold is the sell token
+   * @return The amount of buyToken that was transfered
+   * @dev This function can be frozen using the Freezable interface.
+   */
```

Comment:
nit: how about saying "Deprecated. See comment in sell" instead of duplicating the comment? 

Similar to https://github.com/celo-org/celo-monorepo/pull/4580/files#diff-b3cbc9c000b2a3d8f2f8360587d70368R394

Reply:
I'm sort of wary of doing that because the `@...` comments are used for automatically creating documentation for functions (https://solidity.readthedocs.io/en/develop/natspec-format.html). Any input from @m-chrzan as the resident smart contract expert would be helpful